src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -274,36 +225,34 @@ class BankAccountStep extends React.Component {
-                    <FormScrollView>
-                        <View style={[styles.mh5, styles.mb5, styles.flex1]}>
-                            <AddPlaidBankAccount
-                                text={this.props.translate('bankAccount.plaidBodyCopy')}
-                                onSelect={(params) => {
-                                    this.setState({
-                                        selectedPlaidBankAccount: params.selectedPlaidBankAccount,
-                                    });
-                                }}
-                                onExitPlaid={() => BankAccounts.setBankAccountSubStep(null)}
-                                receivedRedirectURI={this.props.receivedRedirectURI}
-                                plaidLinkOAuthToken={this.props.plaidLinkOAuthToken}
-                                allowDebit
-                            />
-                        </View>
-                        {!_.isUndefined(this.state.selectedPlaidBankAccount) && (
-                            <FormAlertWithSubmitButton
-                                isAlertVisible={Boolean(error)}
-                                buttonText={this.props.translate('common.saveAndContinue')}
-                                onSubmit={this.addPlaidAccount}
-                                message={error}
-                                isLoading={loading}
-                            />
-                        )}
-                    </FormScrollView>
+                    <Form
+                        formID={ONYXKEYS.REIMBURSEMENT_ACCOUNT}
+                        validate={() => ({})}
+                        onSubmit={this.addPlaidAccount}
+                        submitButtonText={this.props.translate('common.saveAndContinue')}
+                        style={[styles.mh5, styles.flexGrow1]}
+                        isSubmitButtonVisible={!_.isUndefined(this.state.selectedPlaidBankAccount)}
+                    >
+                        <AddPlaidBankAccount
+                            onSelect={(params) => {
+                                this.setState({
+                                    selectedPlaidBankAccount: params.selectedPlaidBankAccount,
+                                });
+                            }}
+                            onExitPlaid={() => BankAccounts.setBankAccountSubStep(null)}
+                            receivedRedirectURI={this.props.receivedRedirectURI}
+                            plaidLinkOAuthToken={this.props.plaidLinkOAuthToken}
+                            allowDebit
+                        />
+                    </Form>
                 )}
                 {subStep === CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL && (
-                    <ReimbursementAccountForm
-                        reimbursementAccount={this.props.reimbursementAccount}
+                    <Form
+                        formID={ONYXKEYS.REIMBURSEMENT_ACCOUNT}
```

Comment:
@jayeshmangwani @luacmartins Can we do this: `ONYXKEYS.BANK_FORM: `reimbursementAccount`

Reply:
I think `ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT` would be better